docs/content/getting-started/setup-openfga.mdx

```diff
@@ -65,6 +65,12 @@ If using **Pre-shared key authentication**, you will configure OpenFGA with a se
 If you are going to use this setup in production, you should enable TLS in your OpenFGA server.
 :::
 
+### Playground
+
+The Playground facilitates rapid development by allowing you to visualize and model your application's authorization model(s) and manage relationship tuples with a locally running OpenFGA instance.
+
+The Playground is enabled on port 3000 by default.  To disable the playground, please take a look at [Configure the Server](#configuring-the-server) for more information.
```

Comment:
```suggestion
The Playground is enabled on port 3000 by default. To disable the playground, please take a look at [Configure the Server](#configuring-the-server) for more information.
```

Reply:
"Configure the Server" isn't the section/link of the destination, it's "Configuring the Server". I feel like our links should match the destination they are linking to.